index.html

```diff
@@ -3,89 +3,81 @@
-        </div>        
-        <div class="book">
-            <div class="title">The Hound of the Baskervilles</div>
-            <div class="author">Arthur Conan Doyle</div>
-        </div>        
-        <div class="book">
-            <div class="title">Ulysses</div>
-            <div class="author">James Joyce</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Little Prince</div>
-            <div class="author">Antoine de Saint-Exupéry</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Name of the Rose</div>
-            <div class="author">Umberto Eco</div>
-        </div>        
-        <div class="book">
-            <div class="title">The War of the Worlds</div>
-            <div class="author">H.G. Wells</div>
-        </div>        
-        <div class="book">
-            <div class="title">One Hundred Years of Solitude</div>
-            <div class="author">Gabriel García Márquez</div>
-        </div>        
-        <div class="book">
-            <div class="title">Waiting for Godot</div>
-            <div class="author">Samuel Beckett</div>
-        </div>        
-        <div class="book">
-            <div class="title">Gone with the Wind</div>
-            <div class="author">Margaret Mitchell</div>
-        </div>        
-        <div class="book">
-            <div class="title">Friday</div>
-            <div class="author">Michel Tournier</div>
-        </div>
+    <div class="last-section">
+       <div class="main"> 
+
+      <div class="first-div">
+       <p>The Great Gatsby</p>
+       <p>F. Scott Fitzgerald</p>
+       <p>In Search Of Lost Time</p>
+       <p>Marcel Proust</p>
+       <p>ulysses</p>
+       <p>James Joyce</p>
+       <p>One Hundred Years Of Solitude</p>
+       <p>Gabriel Garcia Marquez</p>
+      </div>   
+    
+       <div class="second-div">
+           <p>The Sound And The Fury</p>
+           <p>William Faulkner</p>
+           <p>To The Lighthouse</p>
+           <p>Virginia Woolf</p>
+           <p>The Little Prince</p>
+           <p>Antoine de Saint-Exupéry</p>
+           <p>Waiting For Godot</p>
+           <p>Samuel Beckett</p>
+       </div>
+
+       <div class="third-div">
+           <p>The Name Of The Rose</p>
+           <p>Umberto Eco</p>
+           <p>Umberto EcoNineteen Eighty-Four</p>
+           <p>George Orwell</p>
+           <p>The Name of the Rose</p>
+           <p>Umberto Eco</p>
+           <p>Gone with the Wind</p>
+           <p>Margaret Mitchell</p>
+       </div>
+
+       <div class="fourth-div">
+        <p>The Magic Mountain</p>
+       <p>Thomas Mann</p>
+       <p>The Hound of the Baskervilles</p>
+       <p>Arthur Conan Doyle</p>
+       <p>The War of the Worlds</p>
+       <p>H.G. Wells</p>
+       <p>Friday</p>
+       <p>Michel Tournier</p>
     </div>
-</div>
+
+     
+     </div>
+   </div>
+ </div>  
+    
+
+
+
 
 
+   <script src ='app.js'></script>
```

Comment:
app.js doesn't exist in the repository.

Reply:
app.js does exist in my local folder of the project but Github does not allow users to upload or push empty files to the platform and my app.js is empty that is why I could not push it or upload it

Reply:
There are some rules in git/github tooling. Like you cannot commit empty directory in git, or push an empty file in github
If you don't use the file how come that you put a placeholder in the file. It is confusing when I read the code.
I expect it but it not there or it is there but does nothing.

Reply:
You are right but honestly I forgot to remove the 'script' tag from html before pushing the code to github.